src/MyEventEmitter.js

```diff
@@ -1,12 +1,64 @@
-'use strict';
-
 export class MyEventEmitter {
-  on() {}
-  once() {}
-  off() {}
-  emit() {}
-  prependListener() {}
-  prependOnceListener() {}
-  removeAllListeners() {}
-  listenerCount() {}
+  constructor() {
+    this._events = {};
```

Comment:
why do use the underscore across-the-code? 🤔 
is there a practical implication?
```suggestion
    this.events = {};
```

Reply:
I wanted to make it private but eslinl does not accept #property so as a old convention I made it _property